### PR TITLE
Add year to past listings #164999194

### PR DESCRIPTION
**Pull request by software-project**
https://www.pivotaltracker.com/story/show/164999194

### CODE REVIEW

File: app/assets/javascripts/customFilters.js.coffee [hunk 1]
```diff
@@ -10,6 +10,15 @@ angular.module('customFilters', [])
     suffix = if relevantDigits <= 3 then suffixes[relevantDigits] else suffixes[0]
     dtfilter + suffix
   ]
+.filter 'dateSuffixOrShort', ['$filter', ($filter) ->
```
Comment (akegan) [inline app/assets/javascripts/customFilters.js.coffee]: is there a better name we could have for this that describes what it's doing?

Comment (akegan) [reply app/assets/javascripts/customFilters.js.coffee]: might also be helpful to add some comments to explain the business rules around this, as per @callaghanc's suggestion recently

Comment (software-project) [reply app/assets/javascripts/customFilters.js.coffee]: I've added a comment and slightly improved the name. I dont have a better idea, but I'm open for suggestions.

**Comment by akegan** [comment]
I'll take this one over and add a test so we can get it merged

**Comment by akegan** [comment]
@chrisdolendo are you up for reviewing the tests that I added to this PR?

**Pull request merged**